framework/framework_initializer_test.go

```diff
@@ -29,9 +30,9 @@ func Test_Framework_InitCtxFunc_AddFunc(t *testing.T) {
 				return ctx, nil
 			},
 			ExpectedOrder: []string{
-				"GetCurrentState",
-				"GetDesiredState",
-				"NewUpdatePatch",
+				"GetCurrentState(deleted=false)",
+				"GetDesiredState(deleted=false)",
+				"NewPatch",
```

Comment:
Now we pass also `(deleted=false|true)` because there is no `NewDeletePatch` and `NewUpdatePatch` anymore. Only `NewPatch`.